templates/BUILD.bazel

```diff
@@ -78,6 +79,7 @@ genrule(
         for d in skeleton \
                  empty-skeleton \
                  create-daml-app \
+                 quickstart-finance \
```

Comment:
You could consider calling it `quickstart-finance-alpha` for now to indicate its early access status.

Reply:
I indicated the early access status in the readme - I'd prefer not to tie the early access status to the command a client run to get this template